transports/msmq/index.md

```diff
@@ -2,16 +2,42 @@
 title: MSMQ Transport
 summary: MSMQ is the primary durable communications technology for Microsoft but does not dynamically detect network interfaces.
 component: MsmqTransport
+related:
+ - samples/msmq/simple
+ - samples/msmq/persistence
+ - samples/msmq/dlq-customcheck
+ - samples/msmq/msmqtosqlrelay
 reviewed: 2016-08-31
 redirects:
  - nservicebus/msmq-information
  - nservicebus/msmq
+tags:
+ - Transport
 ---
 
+## Configuring the endpoint
+
 partial: default
 
 
-## NServiceBus Configuration
+## Advantages and Disadvantages
+
+
+### Advantages
+
+ * MSMQ is natively available as part of the Windows operating system. In Servers, the MSMQ role might need to be turned on.  
+ * MSMQ offers transactional queues which also support distributed transactions. With the transactional behavior, it is easy to get once-only delivery achievable.
```

Comment:
>once-only

Think the accepted term is "exactly once"

Reply:
Nope. It's `exactly-once` :)